### PR TITLE
docs: added nextjs deployment instructions to README, closes #404

### LINKED ISSUE #404: Document how to deploy Next.js website
## Description

/title

## Link to documentation

https://github.com/adobe/spectrum-css

### PR DESCRIPTION
**Pull request by GarthDB**
## How and where has this been tested?
 - **How this was tested:** 
 - **Browser(s) and OS(s) this was tested with:** 

## To-do list

- [x] If my change impacts other components, I have tested to make sure they don't break.
- [x] If my change impacts documentation, I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING document](/.github/CONTRIBUTING.md).

- [x] This pull request is ready to merge.

### CODE REVIEW

**Comment by lazd** [comment]
Excellent, this fills in some gaps.

What is the recommended way of testing the site after `prep`, and before `deploy`?

**Comment by GarthDB** [comment]
There's 2 ways to test it:

1. You can run `npm run dev` any time in the `nextjs` branch to test it out, but it's not a perfect test because it uses a slightly different config
2. You can run `prep` which builds the static site in the `spectrum-css` dir to mimic the url structure on the server (http://opensource.adobe.com/spectrum-css/) and then use any testing server (I like httpster) from the main directory to emulate the server environment keeping the url structure the same (i.e. http://localhost:3333/spectrum-css

**Comment by lazd** [comment]
Alright, but `npm run dev` after `npm run importdata` will give us an up-to-date website to test (latest CSS, latest, `.yml`), yes?

File: README.md [hunk 1]
```diff
@@ -156,12 +156,52 @@ Your `dist/` folder should now have a local copy of the Spectrum CSS docs and mi
...
 Building the project will build and launch the project documentation site in your browser automatically.
 
 See [site generation](site/README.md) and [documentation generation](tools/bundle-builder/docs/README.md) for more information.
 
+#### Generating and deploying external documentation site
+
+Checkout `nextjs` branch and pull if needed. Also dependencies are different in this branch from master so you might need to run `npm install` as well.
+
+```
+git checkout nextjs
```
Comment (lazd) [inline README.md]:
```suggestion
git checkout nextjs
git pull
yarn install
```

**Pull request merged**